lib/galaxy/jobs/runners/util/job_script/DEFAULT_JOB_FILE_TEMPLATE.sh

```diff
@@ -3,6 +3,7 @@
 $headers
 
 _galaxy_setup_environment() {
+    $env_setup_commands
```

Comment:
It'd be probably better to move this down 1 line after `local _use_framework_galaxy="$1"` .

Reply:
Why? Local_use_framework_galaxy should not be overwritten by the user.

Reply:
Because that's the Bash way to bind a function parameter to a variable inside the function. If the `$env_setup_commands` contain the string `$1` that may interfere with that. Doesn't it work for you if moved?

Reply:
You are right, I have moved the statement.